src/quo2/components/loaders/community_card_skeleton/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns quo2.components.loaders.community-card-skeleton.view
```

Comment:
great work here @vkjr. 

Imo it's important that we follow the design system of where and how to get components. In this case the loading variant of community card is within the community card component itself - 
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=912-27100&mode=design&t=V7jJ3BlbZZU2OfGP-4

as such we should just update that component because then it is much easier to find. Unless I am mistaken, here we are making our own mapping with this in `.loaders`. namespace

i.e ideally we add a variant to `quo2.components.community.community-card-view` 

<img width="335" alt="Screenshot 2023-08-01 at 15 47 47" src="https://github.com/status-im/status-mobile/assets/22799766/4a752ebd-8f7d-446a-a36e-12f9cc862ea0">


cc @ilmotta, @ulisesmac

Reply:
yep, I agree. technically it is more a loader :) but yep best to put it there as then it self documents to some degree :)

Reply:
The less exceptions that drift the code from the DS the better, so I think the skeleton as a variant is more correct and will cause less confusion to others. I actually like the fact that loader variants for individual components are in the same Figma page as the component themselves. The `Loaders` page seems to be more related to "loaders for lists of things".